specification/attributes/null_handling.md

```diff
@@ -20,7 +20,8 @@ Indicates how to handle columns that don't have a value.
 ## Requirements
 
 * Columns MUST use NULL when there isn't a value that can be specified for a nullable column.
-* Columns MUST NOT use empty strings or placeholder values such as "Not Set" or "Not Applicable" in columns, regardless of if the column allows nulls or not.
+* String columns MUST NOT use empty strings or placeholder values such as "Not Set" or "Not Applicable" in columns, regardless of whether the column allows nulls or not.
```

Comment:
```suggestion
* Columns MUST NOT use empty strings or placeholder values such as "Not Set" or "Not Applicable" in columns, regardless of whether the column allows nulls or not.
```
If we omit 'String,' the requirement is applicable to Date/Time and other data types?!

Reply:
I question this as there are valid entries that may legitimately be "Not set". Can this be simplified saying that the spec calls for the use of Null to specify no data and does not allow for empty string.

Reply:
@rileyjenk The first bullet in the requirements conveys that null must be used. Is that good enough for what you're looking for? I think if there's a valid scenario for "Not Set", then that wouldn't be a "placeholder", so that should be fine.

Reply:
@rileyjenk @ijurica are you satisfied with the above explanation? If not, can you add a suggestion/comment here?

Reply:
@udam-f2  @ijurica  @flanakin is this now in conflict with the decisions made today? Namely we let the provider specify if they are going to allow nulls or empty string? 

That being said I like this.